api/grpcserver/v2alpha1/account_test.go

```diff
@@ -37,19 +41,26 @@ func TestAccountService_List(t *testing.T) {
 		addr := types.Address{}
 		binary.BigEndian.PutUint64(addr[:], uint64(i))
 
+		template := wallet.TemplateAddress
+		if rand.New(rand.NewSource(int64(i))).Intn(2) == 0 {
+			template = multisig.TemplateAddress
+		}
```

Comment:
It's effectively equivalent to this, isn't it? I mean if you just care that 50% of iterations run with `wallet` and the other half with `multisig`.
```suggestion
		template := wallet.TemplateAddress
		if (i % 2) == 0 {
			template = multisig.TemplateAddress
		}
```